cluster/src/main/java/com/hartwig/pipeline/calling/structural/gridss/stage/Driver.java

```diff
@@ -18,13 +20,11 @@
 import com.hartwig.pipeline.execution.vm.java.JavaJarCommand;
 import com.hartwig.pipeline.resource.ResourceFiles;
 import com.hartwig.pipeline.stages.SubStage;
-import com.hartwig.pipeline.tools.Versions;
 
 public class Driver extends SubStage {
 
-    private static final String GRIDSS_SCRIPT = "gridss.run.sh";
-    private static final String SV_PREP = "sv-prep";
-    private static final String SV_PREP_JAR = "sv-prep.jar";
+    public static final String SV_PREP_DEPTH_ANNOTATION = "com.hartwig.hmftools.svprep.depth.DepthAnnotator";
```

Comment:
these can be package protected rather than public (I think they are only used outside this class in tests)

Reply:
The Gridss unit tests are currently in a different package. Were you also suggesting moving them so as to then meke these protected?

Reply:
My bad, I didn't see that they were in different packages. That said, the package structure here is a little weak (don't really see what the "stage" subpackage adds). Feel free to improve this if you have the urge otherwise you can disregard the comment and I've made a note to come back to this and revisit the packaging.